client/client.go

```diff
@@ -1909,6 +1953,87 @@ func NewAssociateClinicianToUserRequestWithBody(server string, clinicId ClinicId
 	return req, nil
 }
 
+// NewListMembershipRestrictionsRequest generates requests for ListMembershipRestrictions
+func NewListMembershipRestrictionsRequest(server string, clinicId ClinicId) (*http.Request, error) {
+	var err error
+
+	var pathParam0 string
+
+	pathParam0, err = runtime.StyleParamWithLocation("simple", false, "clinicId", runtime.ParamLocationPath, clinicId)
+	if err != nil {
+		return nil, err
+	}
+
+	serverURL, err := url.Parse(server)
+	if err != nil {
+		return nil, err
+	}
+
+	operationPath := fmt.Sprintf("/v1/clinics/%s/membership_restrictions", pathParam0)
+	if operationPath[0] == '/' {
+		operationPath = "." + operationPath
+	}
+
+	queryURL, err := serverURL.Parse(operationPath)
+	if err != nil {
+		return nil, err
+	}
+
+	req, err := http.NewRequest("GET", queryURL.String(), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return req, nil
+}
+
+// NewUpdateMembershipRestrictionsRequest calls the generic UpdateMembershipRestrictions builder with application/json body
+func NewUpdateMembershipRestrictionsRequest(server string, clinicId ClinicId, body UpdateMembershipRestrictionsJSONRequestBody) (*http.Request, error) {
+	var bodyReader io.Reader
```

Comment:
stylistic nitpick, ignorable:
If I'm going to read from something I just encoded/wrote, I usually like to use a `bytes.Buffer` / less temp variables.
```go
var buf bytes.Buffer
err := json.NewEncoder(&buf).Encode(body)
... error handling ...
return NewUpdateMembershipRestrictionsRequestWithBody(server, clinicId, "application/json", &buf)
```